.remarkrc

```diff
@@ -4,6 +4,11 @@
     "preset-lint-markdown-style-guide",
     "preset-lint-recommended",
     "frontmatter",
+    ["remark-lint-ordered-list-marker-value", "ordered"],

```

Comment:
I researched a bit and due to https://github.com/remarkjs/remark-lint/issues/165 those won't affect the presets I think. Or did these work for you?

Anyways, don't spend time on this for v5, can be done afterwards :)